shared/src/vyxal/Globals.scala

```diff
@@ -36,7 +43,7 @@ class Inputs(origInputs: Seq[VAny] = Seq.empty):
 
   /** Uses an array for constant access, not for mutating items */
   private var currInputs = origArr
-  private var allInputs = origInputs.reverse
+  private val allInputs = origInputs.reverse
```

Comment:
This is the best day of my life

Reply:
no.

Reply:
oh wait this isn't a suggestion it's a comment on a change

Reply:
Did you assume I was asking you to change a `var` to a `val` lol